website/client-old/js/controllers/inventoryCtrl.js

```diff
@@ -355,24 +355,19 @@ habitrpg.controller("InventoryCtrl",
     $scope.seasonalShopCategories = Shared.shops.getSeasonalShopCategories(user);
 
     $scope.shouldShowPremiumPetRow = function (potion) {
-      potion = Content.premiumHatchingPotions[potion];
+      premiumPotion = Content.premiumHatchingPotions[potion];
```

Comment:
Damn, how did I miss that!
Just pushed the fix. Thanks for the help and teachings @crookedneighbor and @khipkin .